src/NuGet.Core/NuGet.ProjectModel/Strings.resx

```diff
@@ -127,6 +127,12 @@
     <value>The pack options package type must be a string or array of strings in '{0}'.</value>
     <comment>{0} is the project.json file name.</comment>
   </data>
+  <data name="Invalid_AttributeValue" xml:space="preserve">
```

Comment:
As I scroll this pull request, I've seen this same embedded resource a few times.  Do we have any shared resource we can add this to?

Reply:
Yes there was four and I have removed two but there is two left. One in NuGet.ProjectModel and another one in  NuGet.Commands. Both of these are internal classes. I did not want to add InternalsVisibileTo and create unnecessary dependency between the two.